stdlib/2and3/socket.pyi

```diff
@@ -628,8 +628,10 @@ class socket:
         def getblocking(self) -> bool: ...
     def gettimeout(self) -> Optional[float]: ...
 
-    if sys.platform == 'win32':
-        def ioctl(self, control: object, option: Tuple[int, int, int]) -> None: ...
+    if sys.platform == 'win32' and sys.version_info >= (3, 6):
+        def ioctl(self, control: int, option: Union[int, Tuple[int, int, int], bool]) -> None: ...
```

Comment:
The two branches aren't really necessary because bool is a subtype of int anyway, but I suppose it's better to be explicit.